### ISSUE TITLE
Are the files moved or copied?

### ISSUE THREAD

**Issue by konashevich** [TheLastGimbus/GooglePhotosTakeoutHelper#347]
Сześć TheLastGimbus,

Great script for any Googlers who happen to come across this terrible Takeout, this piece of sh... shame by Google.

One question: Are the files **copied** to the final folder, or are they **moved**?
This is crucial because I need to process 7TB of photos, and it’s a matter of deciding whether I need to buy a couple of SSDs as a buffer.

Thanks)

**Comment by nathan30** [comment]
It's moved :)

**Comment by TheLastGimbus** [comment]
clearly stated in bold in readme: **moves by default**

but you can use `—copy`, see `—help` for details

**Comment by konashevich** [comment]
> clearly stated in bold in readme: **moves by default**
> 
> but you can use `—copy`, see `—help` for details

I hope so, as I encountered a problem during exectution - the terminal closed. I guess it just closed itself (not me!).
I tried to relaunch the process where it left off - when it asked that there is the chosen folder is not empty and what to choose: to delete all files  in it or to continue - I chose to continue. In a couple of seconds the teminal closed again. I tried several times each time was the same. How to fix it?
